src/Models/CsvCatalogItem.js

```diff
@@ -59,20 +65,43 @@ var CsvCatalogItem = function(application, url) {
     this.data = undefined;
 
     /**
-     * Gets or sets the URL from which the {@link CsvCatalogItem#data} was obtained.
-     * @type {String}
+     * Gets or sets the tableStyle object
+     * @type {Object}
      */
-    this.dataSourceUrl = undefined;
+    this.tableStyle = undefined;
 
-    /**
-     * Gets or sets a value indicating whether data points in the CSV are color-coded based on the
-     * value column.
-     * @type {Boolean}
-     * @default true
+   /**
+     * Gets or sets the opacity (alpha) of the data item, where 0.0 is fully transparent and 1.0 is
+     * fully opaque.  This property is observable.
+     * @type {Number}
+     * @default 0.6
      */
-    this.colorByValue = true;
+    this.opacity = 0.6;
 
-    knockout.track(this, ['url', 'data', 'dataSourceUrl', 'colorByValue']);
+    knockout.track(this, ['url', 'data', 'tableStyle', 'opacity', '_clock']);
+
+    knockout.getObservable(this, 'opacity').subscribe(function(newValue) {
+        updateOpacity(this);
+    }, this);
+
+    delete this.__knockoutObservables.clock;
```

Comment:
Maybe I'm missing something, but I don't see why the base class `clock` property needs to be replaced.  Its replacement below looks functionally identical.